Production/plugins/TauTupleProducer.cc

```diff
@@ -950,6 +1030,54 @@ class TauTupleProducer : public edm::EDAnalyzer {
         }
     }
 
+    void FillSV(const TauJet::SVCollection sv_col, const TauJet::PFCandCollection& cands)
+    {
+
+        auto findMatch = [&](const edm::Ptr<reco::Candidate>& recocand) { 
+            for(size_t cand_idx = 0; cand_idx < cands.size(); ++cand_idx) {
+                auto pfcand = dynamic_cast<const reco::Candidate*>(cands.at(cand_idx).candidate);
+                if(pfcand == &(*recocand))
+                   return static_cast<int>(cand_idx);
+            }
+            return -1;
```

Comment:
if TauJet modified as suggested above, no-match situation shouldn't occur. 
```suggestion
            throw std::runtime_error("No SV <-> pfCand match found");
```

Reply:
Do we still actually need this function anyway or filling all SVs from ```selSVs``` is enough ?

Reply:
we need to compute sv_index <-> pf_index associations. If we run only on SV stored in TauJet, such association should always exist (otherwise there is a bug in SV&pf adding code in TauJet)